### PR TITLE
docs: change the default cluster instructions to recommend Kind on Linux

### PR DESCRIPTION
**Pull request by nicks**
Hello @maiamcc,

Please review the following commits I made in branch nicks/ch9397:

5dde607b70ae6b2dbebeda0d491c08f9c09f1863 (2020-09-16 18:36:34 -0400)
docs: change the default cluster instructions to recommend Kind on Linux
Fixes https://github.com/tilt-dev/tilt/issues/3772

Code review reminders, by giving a LGTM you attest that:

* Commits are adequately tested
* Code is easy to understand and conforms to style guides
* Incomplete code is marked with TODOs
* Code is suitably instrumented with logging and metrics

### CODE REVIEW

**Comment by nicks** [comment]
i'm not totally sure this is the right recommendation. i personally still use microk8s every day as my go-to cluster. But I understand the complaints about microk8s, particularly on non-ubuntu platforms.

File: docs/install.md [hunk 1]
```diff
@@ -39,29 +39,15 @@ Linux
...
-```
-
+- Install [Kind with a a local registry](https://github.com/tilt-dev/kind-local/blob/master/README.md)
 - Install `tilt` with:
 
 ```bash
 curl -fsSL https://raw.githubusercontent.com/tilt-dev/tilt/master/scripts/install.sh | bash
 ```
 
+Alternatively, Ubuntu users sometimes prefer [Microk8s](choosing_clusters.html#microk8s) instead of Kind.
```
Comment (maiamcc) [inline docs/install.md:49]: is it worth it to note here WHY you might go for microk8s over Kind?

**Review by maiamcc — APPROVED** [review]
ulllp! well, worth a shot!

**Comment by maiamcc** [comment]
looks like we're still recommending D4D for MacOS, how do you feel about changing that rec to be Kind as well? I think it could also be good to put a tl/dr "our rec for your OS" at the top.

lmk if you want me to do the above, or if you want to do it, either is fine with me

File: docs/install.md (hunk 1 above)
Comment (nicks) [reply docs/install.md:49]: done

**Comment by nicks** [comment]
re: D4D - this is a way outside the scope of the original user complaint linked above, but I'll take the bait:

Different docs have different audiences.

In the install guide, the audience is trying to figure out how to get Tilt setup as quickly as possible. For that audience, I think D4D is 100% the right recommendation.

But the audience of a 10 minute video about how to choose a local dev cluster is a different audience, and might be a different/more nuanced recommendation.

File: docs/install.md [hunk 2]
```diff
@@ -39,29 +39,18 @@ Linux
...
-- Make microk8s your local Kubernetes cluster:
-
-```bash
-sudo microk8s.kubectl config view --flatten > ~/.kube/microk8s-config && \
-KUBECONFIG=~/.kube/microk8s-config:~/.kube/config kubectl config view --flatten > ~/.kube/temp-config && \
-mv ~/.kube/temp-config ~/.kube/config && \
-kubectl config use-context microk8s
-```
-
+- Install [Kind with a a local registry](https://github.com/tilt-dev/kind-local/blob/master/README.md)
```
Comment (ellenkorbes) [inline docs/install.md:42]: > Kind with a a local registry

Typo!

**Comment by ellenkorbes** [comment]
Also might be worth noting for the Windows install that those commands should be typed on Powershell, not on CMD.

File: docs/install.md (hunk 2 above)
Comment (nicks) [reply docs/install.md:42]: :grimacing: https://github.com/tilt-dev/tilt.build/pull/748

**Pull request merged**